redis/src/test/java/org/apache/calcite/adapter/redis/RedisDataCaseBase.java

```diff
@@ -99,8 +99,6 @@ public void makeData() {
 
   @AfterEach
   public void shutDown() {
-    if (null != pool) {
```

Comment:
I don't think so. When we init the pool throw exception, if, remove the condition, the code will throw NPE.

Reply:
@NobiGo Thanks for reviewing, sorry for my mistake, @AfterEach will still execute if @BeforeEach fails. I have fixed it.